codegen/smithy-dafny-codegen-test/src/test/java/software/amazon/polymorph/smithyjava/JavaTestModels.java

```diff
@@ -40,8 +41,14 @@ class JavaTestModels extends TestModelTest {
     DISABLED_TESTS.add("Union");
     DISABLED_TESTS.add("aws-sdks/kms-lite");
     DISABLED_TESTS.add("aws-sdks/sqs");
+    DISABLED_TESTS.add("aws-sdks/sqs-via-cli");
+    //TODO: Add support for Recursive shapes.
+    DISABLED_TESTS.add("RecursiveShape");
     // S3 is not yet supported
     DISABLED_TESTS.add("aws-sdks/s3");
+
+    //TODO: https://github.com/smithy-lang/smithy-dafny/issues/599
+    DISABLED_TESTS.add("Positional");
```

Comment:
Note to self: only necessary because #735 will make this not work for some languages